src/frontend/komponenter/Fagsak/useFagsakApi.ts

```diff
@@ -103,8 +103,16 @@ const useFagsakApi = (
         }
 
         settSenderInn(true);
+
+        const aktivBehandling = fagsak.behandlinger.find(b => b.aktiv);
+        const resutat =
```

Comment:
Would it be better to do this logic backend?

Reply:
Not sure. We can change it when it is necessary.